template/11-monetization.md

```diff
@@ -1,6 +1,13 @@
 # Business Model
 
-*Expected operating Costs*
+### Expected operating cost
+The operating costs of our app are proportional to the number of users. The choice of Supabase allows us to avoid vendor lock-in if it becomes too costly to pay for the database and storage. The pricing page of Supabase shows us that 25$ per month is sufficient for 100'000 monthly active users. Then the cost per user increases, but thanks to the fact that Supabase can be deployed on our own hardware, we can keep the cost down. For example, if we decide to deploy Supabase on a virtual private server, we estimate that it will cost only 0.00015$ per month per monthly active user. We clearly see that these operating costs are small.
 
-*Revenue Streams*
+### Revenue stream
+The revenue stream of our app comes from mutliples sources.  
+First, we can collect some group behavior data from our users. For example, which categories of events are the most successful, or what's the best location for an event to be successful. With these insights on our users, we can introduce an optional monthly subscription of $5 for the associations which want to improve their chances to organize a successful event.  
+Second, we can develop themed versions of the app for special occasions. For example a week-long event on a big area with a lot of events in parallel, like CES, could ask us to create a special app for the occasion that would allow the participants to have an overview of all the events, join some of them and find them easily on the map. This would occasionally bring us a big income, probably about 10'000$ per mission.  
+Third, we can monetize some services in our application, which allows the associations to have easy integration of all the things that come with the organization of an event. For example, a ticketing system can easily be added to our app. The user could buy a ticket for a paid event without needing another app or website. The associations would have some management interface with the participant to allow easy verification, refunds, and more. We just take a small fee on the ticket price, for example, 5%.
 
+### Extrapolation
+In total, we estimate that if our app becomes successful enough to be adopted by 1'000'000 users, with 200 associations that pay for premium features and 5'000'000$ worth of tickets per year, we can estimate a minimal cost of 15$ per month and a revenue of about 252'000$ per year. This gives us a consistent income that will allow us to develop more features for our app.
```

Comment:
Similarly, here it would be better to either explain the numbers or provide a more general idea of extrapolation such as after we generate a significant revenue after selling to our app to multiple universities in Europe, we could hire more developers who would be responsible for further development and maintenance of the app.